src/main/java/ay2122s1_cs2103t_w16_2/btbb/logic/parser/util/CliSyntax.java

```diff
@@ -16,6 +16,7 @@
     public static final Prefix PREFIX_INGREDIENT_QUANTITY_TO = new Prefix("iqt/");
     public static final Prefix PREFIX_INGREDIENT_UNIT = new Prefix("iu/");
     public static final Prefix PREFIX_ORDER_DEADLINE = new Prefix("od/");
+    public static final Prefix PREFIX_ORDER_IS_DONE = new Prefix("of/");
```

Comment:
Since the prefix is `of/` i think its better to name the variable as isFinished in the model